autonomy/analyse/service.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # ------------------------------------------------------------------------------
 #
-#   Copyright 2023 Valory AG
+#   Copyright 2023-2024 Valory AG
```

Comment:
Why the change?

Reply:
This happened after running generators. Too many files changed.